dashboard/modules/reporter/reporter_agent.py

```diff
@@ -189,12 +207,48 @@ def jsonify_asdict(o) -> str:
         "MB",
         ["ip"],
     ),
+    "workers_mem_shm": Gauge(
+        "workers_mem_shm",
+        "SHM usage of all workers on the node. Only available on Linux.",
+        "MB",
+        ["ip"],
+    ),
     "workers_mem_uss": Gauge(
         "workers_mem_uss",
         "USS usage of all workers on the node. Only available on Linux",
         "MB",
         ["ip"],
     ),
+    "workers_mem_pss": Gauge(
+        "workers_mem_pss",
+        "PSS usage of all workers on the node. Only available on Linux",
+        "MB",
+        ["ip"],
+    ),
+    "agent_cpu": Gauge(
+        "agent_cpu",
+        "Total CPU usage of an agent on a node.",
+        "percentage",
+        ["ip"],
+    ),
+    "agent_mem": Gauge(
+        "agent_mem",
```

Comment:
I think it is better not to label RSS / USS / pss because the sum of different labels wouldn't match the sum of total metrics. So I just created separate metrics.

Reply:
How about having `process_memory_uss/rss/pss` metrics and setting the process name as a label? This could generalize to other process names such as the worker task names as well in the future.

Reply:
ah yeah I think that's a great idea. I will handle that here. btw is it okay if I call it just component name = {"raylet", "workers", "agent", "dashboard"}?

Reply:
and no component ID (I think it is hard to obtain things like worker ID, and using pid is probably a bad idea)

Reply:
Component sounds good (we already have a "Component" label it seems).

Reply:
it's done!